src/test/rpc/Book_test.cpp

```diff
@@ -800,6 +800,460 @@ class Book_test : public beast::unit_test::suite
             books)[jss::status] == "success");
     }
 
+    void testTrackOffers(void)
+    {
```

Comment:
@bachase Yes, our usual convention is to put `testcase` at the beginning of each test. Some tests may have been overlooked (or maybe there's a reason they're missing - I don't know).